app/services/proofing/lexis_nexis/request.rb

```diff
@@ -51,9 +53,24 @@ def base_url
       end
 
       def build_request_headers
-        {
+        headers = {
           'Content-Type' => 'application/json',
         }
+        headers['Authorization'] = hmac_authorization if hmac_auth_enabled?
+        headers
+      end
+
+      def hmac_auth_enabled?
+        IdentityConfig.store.lexisnexis_hmac_auth_enabled
+      end
+
+      # Example HMAC auth header from RDP_REST_V3_DecisioningGuide_March22.pdf, page 21
```

Comment:
ditto re: "example"
```suggestion
      # HMAC auth header from RDP_REST_V3_DecisioningGuide_March22.pdf, page 21
```

Reply:
Actually, this comment can be removed - the code it referenced moved to RequestSigner